settings.gradle

```diff
@@ -0,0 +1,42 @@
+pluginManagement {
+    plugins {
+        // Gradle is awkward about declaring versions for plugins
+        // Specifying it here, rather than build.gradle, makes it possible
+        // to include the extension as a subproject of QuPath itself
+        // (which is useful during development)
+        id 'org.bytedeco.gradle-javacpp-platform' version '1.5.8'
+    }
+}
+
+// TODO: Name your QuPath extension here!
+rootProject.name = 'my-qupath-extension'
```

Comment:
Should we modify it to `qupath-extension-sam`?

Reply:
Oh yes, we definitely should! Copied too much from the template...

Do you want that name? I started the `qupath-extension-something` convention just to help me distinguish by own repos and help organize them alphabetically, but there is not really any need for others to follow it...

Reply:
I got it, thanks. I will keep the name `qupath-extension-sam`.